packages/mobile/src/transactions/__snapshots__/TransferConfirmationCard.test.tsx.snap

```diff
@@ -1672,7 +1672,7 @@ exports[`TransferConfirmationCard renders correctly for sent escrow transaction
                 >
                   
                   $
-                  0.001
+                  0
```

Comment:
Confirming that this change is expected as we now only round up to 0.001 if it is non zero?

Reply:
Well, this is tough question. When fee is precisely zero, do we want still to show `0.001`? cc: @cmcewen @jmrossy @jeanregisser

Reply:
After talking with @cmcewen I decided the following: fees should not be zero in the long run, but now we have tobin tax set to zero and transaction fee is not present in the blockchain api for exchanges. @nityas also has agreed to if the value is exactly `0` - show zero. In the future these fees should be non-zero. Let me know what do you think?

Reply:
Sgtm!